source/_integrations/gree.markdown

```diff
@@ -11,14 +11,53 @@ ha_codeowners:
 ha_domain: gree
 ---
 
-The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/).
+The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/) in Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Configuration](#configuration)
+- [Supported models](#supported-models)
```

Comment:
these are no platform types 😄

Reply:
VS Code markdown plugin is mangling this on save. Anyway, fixed now.